akka-diagnostics/src/main/scala/akka/diagnostics/ConfigChecker.scala

```diff
@@ -555,6 +556,29 @@ class ConfigChecker(system: ExtendedActorSystem, config: Config, reference: Conf
       else Nil
     }
 
+  private def checkInternalDispatcherSize(): List[ConfigWarning] =
+    ifEnabled("internal-dispatcher-size") { checkerKey =>
+      val path = internalDispatcherPath
+
+      val size = dispatcherPoolSize(config.getConfig(path))
```

Comment:
It is also possible to reconfigure it to be the default dispatcher, for small footprint, in that case it's already covered, so maybe an extra if-else check for `akka.actor.internal-dispatcher = akka.actor.default-dispatcher` and do nothing if it is replaced by the default one?

Reply:
added